codegen.json

```diff
@@ -69,6 +70,7 @@
           "UpdateDimensionFieldSuccess": "./types/UpdateDimensionFieldSuccess#UpdateDimensionFieldSuccessSource",
           "UpdateGitLabDimensionFieldSuccess": "./types/UpdateGitLabDimensionFieldSuccess#UpdateGitLabDimensionFieldSuccessSource",
           "UpdateMeetingPromptSuccess": "./types/UpdateMeetingPromptSuccess#UpdateMeetingPromptSuccessSource",
+          "UpdateSAMLSuccess": "./types/UpdateSAMLSuccess#UpdateSAMLSuccessSource",
```

Comment:
-1 no need to add mapping if there's no such file as graphql codegen will generate incorrect types

Reply:
@BartoszJarocki  this was  autogenerated by `codegen` at the point of creating this `mutation`,  I would appreciate your input on whether or not it is necessary to remove it?